### PR TITLE
Add plotly chart-studio package

### PR DESCRIPTION
**Pull request by dhirschfeld**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/chart-studio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - chart_studio
+
+about:
+  home: https://github.com/plotly/plotly.py
+  license: MIT
+  license_family: MIT
+  license_file: "/packages/python/chart-studio/LICENSE.txt"
```
Comment (npavlovikj) [inline recipes/chart-studio/meta.yaml]: This file doesn't exist in the pypi tarball. Please add the License file from the repo (https://github.com/plotly/plotly.py/blob/master/LICENSE.txt), and change license_file to `license_file: LICENSE.txt`.

File: recipes/chart-studio/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  summary: "This package contains utilities for interfacing with Plotly's Chart Studio service"
+  doc_url: https://help.plot.ly/tutorials/#fundamentals
+  dev_url: https://github.com/plotly/plotly.py/tree/master/packages/python/chart-studio
+
+extra:
+  recipe-maintainers:
+    - jonmmease
+    - timkpaine
+    - dhirschfeld
+
```
Comment (npavlovikj) [inline recipes/chart-studio/meta.yaml]: Please remove empty lines.

**Comment by dhirschfeld** [comment]
@timkpaine, @jonmmease - are you happy being added as maintainers of this `conda-forge` package?

**Comment by jonmmease** [comment]
Sure, also @nicolaskruchten and @emmanuelle. Thanks!

**Comment by timkpaine** [comment]
Sure

**Comment by npavlovikj** [comment]
The recipe LGTM! @dhirschfeld , do you want to add the suggested two maintainers and check with them if they are willing to be be listed here? Either way, I am good with merging this recipe.

**Comment by dhirschfeld** [comment]
I've added both as maintainers...

**Comment by npavlovikj** [comment]
Thanks @dhirschfeld !

**Comment by dhirschfeld** [comment]
Thanks for the review & merge @npavlovikj!

**Pull request merged**